.azure-pipelines/windows.sh

```diff
@@ -61,15 +61,12 @@ clone_repos
 ################################################################################
 
 if [ "$MODEL" == "64" ] ; then
-    export MODEL_FLAG="-m64"
     MAKE_FILE="win64.mak"
     LIBNAME=phobos64.lib
 elif [ "$MODEL" == "32mscoff" ] ; then
-    export MODEL_FLAG="-m32mscoff"
     MAKE_FILE="win64.mak"
     LIBNAME=phobos32mscoff.lib
 else
-    export MODEL_FLAG="-m32"
     export LIB="$PWD/dmd2/windows/lib"
```

Comment:
Not sure why this is still necessary, but linking the druntime unittests failed without this (Optlink complaining about incompatible libs in MSVC libdirs). So somehow the optlink.exe used for that command seems to respect the LIB env var.